leku/src/main/java/com/schibstedspain/leku/LocationPickerActivity.java

```diff
@@ -723,7 +722,7 @@ private void startVoiceRecognitionActivity() {
       try {
         startActivityForResult(intent, REQUEST_PLACE_PICKER);
       } catch (ActivityNotFoundException e) {
-        Log.d(LocationPickerActivity.class.getName(), e.getMessage());
+        setTracking(TrackEvents.startVoiceRecognitionActivityFailed);
```

Comment:
@rocboronat You're right. `track` express better the intent of what it does. I'm going to change it. Thanks for the suggestion!